front/app/containers/IdeasShow/components/AIInsights/index.tsx

```diff
@@ -0,0 +1,125 @@
+// This code is a prototype for input authoring. Clean-up will follow after the prototype phase.
+import React, { useEffect, useState } from 'react';
+
+import {
+  Box,
+  Button,
+  Text,
+  Title,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+import useAddAuthoringAssistance from 'api/authoring_assistance/useAuthoringAssistance';
+
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+import IdeaCard from 'components/IdeaCard';
+import TextArea from 'components/UI/TextArea';
+
+import { useIntl } from 'utils/cl-intl';
+
+import messages from './messages';
+
+const AIInsights = ({ ideaId }: { ideaId: string }) => {
```

Comment:
I might be tempted to name this `AuthoringAssistanePrototype` instead so it's quite clear for now